.gitignore

```diff
@@ -38,6 +38,7 @@ Makefile
 /include/swtpm.h
 /man/man3/*.3
 /man/man8/*.8
+!/man/man8/swtpm-localca.8
```

Comment:
The `swtpm-localca(8)` manual page is now static and contains a `.so` alias for `swtpm_localca(8)`. Since it's no longer an auto-generated file, it must be versioned and we don't want Git ignoring it.